media-source/mediasource-util.js

```diff
@@ -330,16 +330,15 @@
     };
 
     window['MediaSourceUtil'] = MediaSourceUtil;
-    window['media_test'] = function(testFunction, description, options)
+    window['media_test'] = function(testFunction, description)
```

Comment:
@qiuzhong can you separate out just this change? This is one that we definitely should do, since the options aren't used anywhere.

Reply:
I reverted the `media_test` function.

Reply:
I mean that these are changes that we should land whether or not we remove properties entirely, so submitting those as a separate PR would be great.

Reply:
Sorry, I misunderstood your meaning. This time I reverted the whole test file and  I'll file a new PR for it.